compose.yml

```diff
@@ -0,0 +1,63 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile.libp2p
+    ports:
+      - "3000:3000/tcp"
+      - "8080:8080/tcp"
+    volumes:
+      - ./testnet:/etc/nomos
+    environment:
+      - NET_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY}
+    entrypoint: /etc/nomos/run_bootstrap.sh
+
+  libp2p-node:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile.libp2p
+    volumes:
+      - ./testnet:/etc/nomos
+    deploy:
+      replicas: ${DOCKER_COMPOSE_LIBP2P_REPLICAS} 
+    depends_on:
+      - bootstrap
+      - etcd
+    environment:
+      - DOCKER_REPLICAS=${DOCKER_COMPOSE_LIBP2P_REPLICAS}
+      - ETCDCTL_ENDPOINTS=${DOCKER_COMPOSE_ETCDCTL_ENDPOINTS}
+      - ETCDCTL_API=${DOCKER_COMPOSE_ETCDCTL_API}
+      - NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK}
+      - OVERLAY_NODES=${DOCKER_COMPOSE_OVERLAY_NODES}
+      - NET_INITIAL_PEERS=${DOCKER_COMPOSE_NET_INITIAL_PEERS}
+    entrypoint: /etc/nomos/run_libp2p.sh
+
+  mixnet-node:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile.mixnet
```

Comment:
This file is in no way different from `Dockerfile.libp2p`. What's the point of it?

Reply:
Moved `Dockerfile.libp2p/mixnet` -> `testnet/Dockerfile`. The image should contain everything needed for spawning different nodes + additional services like etcd.